package/tests/testCPU.py

```diff
@@ -38,13 +40,14 @@ def test_runTestWithParams(self):
             self.assertEqual(prmon_rc, 0, "Non-zero return code from prmon")
             prmonJSON = json.load(open("prmon.json"))
```

Comment:
I just noticed this w/ a warning on my Docker/CentOS7 test. I believe we don't close this json file:

```
./testCPU.py:41: ResourceWarning: unclosed file <_io.TextIOWrapper name='prmon.json' mode='r' encoding='ANSI_X3.4-1968'>
  prmonJSON = json.load(open("prmon.json"))
```

Small point but would be good to close it properly (I think applies to other scripts as well).

Reply:
I wonder why I never saw that myself? Usually I just let python files close themselves as they go out of scope, but this might be better as:

```python
with open("prmon.json") as jsonfh:
    prmonJSON = json.load(jsonfh)
    ...
```

We should create an issue to track this.